### PR TITLE
add app-title

### PR DESCRIPTION
**Pull request by gitboej**
In this PR i created the title for the quiz-app

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
```
Comment (aaaleeenaaa) [inline styles.css:4]: Maybe you want to add *::before and *::after here?

**Review by KivancUlucan — APPROVED** [review]
Super

**Review by aaaleeenaaa — CHANGES_REQUESTED** [review]

File: styles.css (hunk 1 above)
Comment (gitboej) [reply styles.css:4]: Done, thanks for your feedback 👍

**Review by aaaleeenaaa — APPROVED** [review]
Looks great! :)

**Pull request merged**